### ISSUE TITLE
MD013: Line-specific disable rule configuration ignored when overlong line elsewhere is reformatted

### ISSUE THREAD

**Issue by sisp** [rvben/rumdl#501]
### rumdl version

v0.1.42

### Installation method

cargo install

### Bug description

When an overlong line is reformatted somewhere in a Markdown document, a line-specific configuration that disables the `MD013` rule for an overlong line is ignored.

### Steps to reproduce

1. Create a file `test.md` with this content:

    ```markdown
    # Test

    <!-- rumdl-disable MD013 -->
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    <!-- rumdl-enable MD013 -->

    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    ```

1. Run `rumdl check --fix test.md`

1. See `test.md` now and observe that both overlong lines are reformatted.

### Expected behavior

The overlong line for which the `MD013` rule is disabled should remain unchanged:

```diff
 # Test

 <!-- rumdl-disable MD013 -->
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
 <!-- rumdl-enable MD013 -->

-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua.
```

### Actual behavior

The overlong line for which the `MD013` rule is disabled is reformatted.

```diff
 # Test

 <!-- rumdl-disable MD013 -->
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua.
 <!-- rumdl-enable MD013 -->

-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua.
```

### Configuration

```toml
[global]
line-length = 80

[MD013]
reflow = true
```

### Operating system

Linux

### Additional context

The bug exists also for the same-line, next-line and capture/restore comments.

**Comment by rvben** [comment]
Thanks for reporting this, @sisp. Fixed in v0.1.43 — inline disable comments are now respected during `--fix` mode. Closing as resolved.

**Comment by sisp** [comment]
Thank you very much for the super quick fix, @rvben! :bow:
